flake.nix

```diff
@@ -42,6 +42,13 @@
     treefmt-nix.url = "github:numtide/treefmt-nix";
     treefmt-nix.inputs.nixpkgs.follows = "nixpkgs";
 
+    # Pict-rs is the image hosting server for Lemmy.
+    # 0.4 comes with a separation of database and file storage which we require but isn't
+    # in a stable release yet.
+    # After pict-rs 0.4 is in nixpkgs we can remove this input.
```

Comment:
When is this going to be done?

Reply:
Untested: https://github.com/NixOS/nixpkgs/pull/243218

Reply:
I also had a WIP branch for the same thing but I didn't know how to tackle the schema migration from 0.3 to 0.4 in the NixOS module.

Reply:
I've done a more comprehensive job in https://github.com/NixOS/nixpkgs/pull/243373.